doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+
+.. important::
+   |NCS| v1.1.0 does not provide support for nRF5340.
+   For nRF5340 support, check out the master branch.
+
+Highlights
+**********
+
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.1.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported modem firmware
+************************
+
+* mfw_nrf9160_1.1.0
+
+Use the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v9.8.1
+     - `nRF Command Line Tools`_
+   * - nRF Connect for Desktop
+     - v3.0.0 or later
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.20a or later.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+???
+===
+
+(? where should these go? where are they defines/documented? what do they do?)
+
+(new) PAW3212 driver
+(new) nrf52810_pca20045 board
+(new) nrf52_pca20044 board
+
+adp536x driver:
+
+    Added buck discharge resistor configuration.
+
+
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`lwm2m_carrier` - demonstrates how to use the :ref:`liblwm2m_carrier_readme` library to connect to the operator LwM2M network.
+  * :ref:`lwm2m_client` - demonstrates how to use LwM2M to connect an nRF9160 DK to an LwM2M server such as Leshan Demo Server via LTE.
```

Comment:
I suggest that we rephrase this slightly to make it clearer that this sample is using Zephyr's upstream LwM2M stack to implement a sample LwM2M client application; you can't connect to the operator network with this, but only run the upstream LwM2M implementation against a demo server.

Reply:
Updated to:
  * :ref:`lwm2m_client` - demonstrates how to use Zephyr's :ref:`lwm2m_interface` to implement a sample LwM2M application.
     This sample can run against an LwM2M demo server, but cannot connect to the operator network.